src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -0,0 +1,74 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkPIDController;
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class Shooter extends Command {
+ private CANSparkMax motor1;
+ private CANSparkMax motor2;
+
+ private RelativeEncoder encoder1;
+ private RelativeEncoder encoder2;
+ private SparkPIDController pidController1;
+ private SparkPIDController pidController2;
+ private double KP =0;
+ private double KD=0;
+ private double KI=0;
+ public static double speakerSPEED = 4000;
+ public static  double ampSPEED = 2000;
+
+ public Shooter(){
+     motor1 = new CANSparkMax(12, CANSparkLowLevel.MotorType.kBrushless);
+     motor2 = new CANSparkMax(13, CANSparkLowLevel.MotorType.kBrushless);
+     pidController1 = motor1.getPIDController();
+     pidController2 = motor2.getPIDController();
+     pidController1.setP(KP);
+     pidController1.setI(KI);
+     pidController1.setD(KD);
+     pidController2.setP(KP);
+     pidController2.setI(KI);
+     pidController2.setD(KD);
```

Comment:
Don't provide the same PID variables for 2 different PIDs even if temporary. And why why is your `ampSPEED` and `speakerSPEED` public and also not final? As well as since both variables are static and final you should name them `AMP_SPEED` and `SPEAKER_SPEED` instead of what they currently are.